docs/content/Avatar.mdx

```diff
@@ -5,34 +5,49 @@ componentId: avatar
 source: https://github.com/primer/react/blob/main/src/Avatar.tsx
 ---
 
-import {Avatar} from '@primer/components'
+import {Avatar, Box} from '@primer/components'
 
-```jsx live
-<Avatar src="https://avatars.githubusercontent.com/primer" />
+```js
+import {Avatar} from '@primer/components'
 ```
 
-## Props
+## Examples
 
-<Props of={Avatar} />
+### Default
 
-## Examples
+```jsx live
+<Avatar src="https://avatars.githubusercontent.com/primer" />
+```
 
 ### Square
 
 ```jsx live
 <Avatar square src="https://avatars.githubusercontent.com/primer" />
 ```
 
-### AvatarPair
+## Props
 
-```jsx live
-<AvatarPair>
-  <Avatar src="https://avatars.githubusercontent.com/github" />
-  <Avatar src="https://avatars.githubusercontent.com/primer" />
-</AvatarPair>
-```
+### Avatar
+
+<PropsTable>
+  <PropsTableRow name="src" type="string" required description="URL of the avatar image." />{' '}
+  <PropsTableRow
+    name="alt"
+    type="string"
+    defaultValue="''"
```

Comment:
❗ I'm surprised this is `''` and not undefined.

Reply:
I think this is intentional. `alt=''` is treated differently than an undefined `alt`. From https://davidwalsh.name/accessibility-tip-empty-alt-attributes:

> Images with only visual value should have an empty alt attribute set on them.  Omitting the alt attribute makes most screen readers read out the entire image URL and providing an alt attribute when the image is for visual purposes only is just as useless.

Reply:
TIL, thanks!